disnake/user.py

```diff
@@ -456,6 +468,75 @@ async def edit(
         return ClientUser(state=self._state, data=data)
 
 
+class Nameplate:
+    """
+    Represents the decoration behind the name of a user that appears
+    in the server, DM and DM group members list.
+
+    .. versionadded:: 2.11
+
+    Attributes
+    ----------
+    sku_id: :class:`int`
+        The ID of the nameplate SKU.
+    label: :class:`str`
+        The label of this nameplate.
+    palette: :class:`str`
+        The background color of the nameplate.
+    """
+
+    __slots__ = (
+        "sku_id",
+        "label",
+        "palette",
+        "_state",
+        "_asset",
+    )
+
+    def __init__(self, *, state: ConnectionState, data: NameplateDataPayload) -> None:
+        self._state: ConnectionState = state
+        self.sku_id: int = int(data["sku_id"])
+        self._asset: str = data["asset"]
+        self.label: str = data["label"]
+        self.palette = data["palette"]
+
+    @property
+    def nameplate_animated_asset(self) -> Asset:
```

Comment:
```suggestion
    def animated_asset(self) -> Asset:
```
maybe? or just `animated`?

Reply:
https://github.com/DisnakeDev/disnake/pull/1303/commits/6b43e0c477a308bb45bee7c1442df50748c976a8